src/react-apps/applications/runtime/src/features/instantiate/containers/PartySelection.tsx

```diff
@@ -182,12 +181,12 @@ function PartySelection(props: IPartySelectionProps) {
     if (!language || !language.party_selection) {
       return null;
     }
-    if (location.state !== undefined &&
-      (location.state as IRedirectReason) &&
-      (location.state as IRedirectReason) !== undefined) {
-        switch ((location.state as IRedirectReason).errorType) {
+    if ((match.params as IRedirectParams).errorCode !== null) {
```

Comment:
Vil dette alltid være bli et objekt? match.params altså.
Hvis match.params noensinne har en sjanse for å være null så vil det krasje her

Reply:
`match.params` will always be an object. Since it is supplied from the `withRouter` HoC. So params will be an empty object if you're on a route that has a specified a param.